### PR TITLE
Added a bugs section and added the bug in pd.merge() 

### PR DESCRIPTION
**Pull request by umeshG34**
1. Made minor spelling corrections.
2. Added a `7. Bugs` section and added `7.1  pd.merge() joins over np.nan and None` as the first bug.

Please review and let me know of any changes you would like me to make. 
Thank you!

### CODE REVIEW

File: just-pandas-things.ipynb [hunk 1]
```diff
@@ -163,7 +163,7 @@
     }
    ],
    "source": [
-    "import pandas as pd\n",
+    "import numpy as np, pandas as pd\n",
```
Comment (chiphuyen) [inline just-pandas-things.ipynb:166]: let's import `np` only when it's used (e.g. in your added section on merge bugs)

File: just-pandas-things.ipynb [hunk 2]
```diff
@@ -535,12 +535,12 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/Users/chip/miniconda3/envs/py3/lib/python3.6/site-packages/ipykernel_launcher.py:2: SettingWithCopyWarning: \n",
+      "<ipython-input-12-fa76da25894c>:2: SettingWithCopyWarning: \n",
```
Comment (chiphuyen) [inline just-pandas-things.ipynb:538]: let's not re-run cells you didn't change so the results aren't overwritten

File: just-pandas-things.ipynb [hunk 3]
```diff
@@ -3360,6 +3360,308 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "---\n",
+    "## 7. Bugs\n",
```
Comment (chiphuyen) [inline just-pandas-things.ipynb:3364]: let's call this `Known bugs` and mention the links to the tracking PRs earlier in the post so people know that it's a well-known bug and not just a personal opinion.

File: just-pandas-things.ipynb [hunk 4]
```diff
@@ -3360,6 +3360,308 @@
...
+       "4     B     2     B     22"
+      ]
+     },
+     "execution_count": 41,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df1 = pd.DataFrame({'col1' : [np.nan, 'B', 'C', None], 'col2' : [1, 2, 3, 4]})\n",
```
Comment (chiphuyen) [inline just-pandas-things.ipynb:3581]: let's use `"` instead of `'` for consistency in the entire notebook

File: just-pandas-things.ipynb [hunk 5]
```diff
@@ -3360,6 +3360,308 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "---\n",
+    "## 7. Bugs\n",
+    "### 7.1 `pd.merge()` joins over np.nan and None\n",
```
Comment (chiphuyen) [inline just-pandas-things.ipynb:3365]: nit: escape `np.NaN` and `None` for consistency

**Review by chiphuyen — CHANGES_REQUESTED** [review]
Thanks for contributing! There are several comments in the PR, but here are two major ones:

1. Can you make it into 2 separate PRs since they do different things: one for typos and the other for adding the known bugs?
2. Try not to re-run cells that you didn't update to avoid overwriting the existing results.

Cheers,

File: just-pandas-things.ipynb (hunk 1 above)
Comment (umeshG34) [reply just-pandas-things.ipynb:166]: Done in PR#9

File: just-pandas-things.ipynb (hunk 2 above)
Comment (umeshG34) [reply just-pandas-things.ipynb:538]: Made sure of this in PRs #8 and #9. I apologize. I hadn't checked the diff until after your comment.

File: just-pandas-things.ipynb (hunk 3 above)
Comment (umeshG34) [reply just-pandas-things.ipynb:3364]: Done in #9

File: just-pandas-things.ipynb (hunk 4 above)
Comment (umeshG34) [reply just-pandas-things.ipynb:3581]: done in #9 .Thank you Chip

**Comment by umeshG34** [comment]
Splitting this PR into #8 and #9 . All the feedback comments done there. Thank you for taking the time to review this Chip!

**Pull request closed without merging**